contracts/Coordinator.sol

```diff
@@ -1,41 +1,41 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.4;
+pragma solidity 0.8.13;
```

Comment:
Should we update the other contracts and hardhat config as well? I pushed a branch with those changes, we can just open the PR if it's useful https://github.com/thisisarchimedes/Archimedes_Finance/compare/main...cm-update-solidity